pkg/registry/meta.go

```diff
@@ -230,8 +230,8 @@ func (r *Resource) scrapePrelude(doc *html.Node, preludeXPath string) error {
 	}
 	r.Description = strings.TrimSpace(strings.Replace(r.Description, "|-", "", 1))
 
-	if r.SubCategory == "" || r.Title == "" {
-		return errors.Errorf("failed to parse prelude. Description: %s, Subcategory: %s, Title name: %s. Raw data:%s\n",
+	if r.Title == "" {
```

Comment:
@ulucinar Github provider doesn't have any subcategory. Is this change enough to accommodate such providers?